cvprac/cvp_client.py

```diff
@@ -734,10 +740,8 @@ def _make_request(self, req_type, url, timeout, data=None,
                                ' response data. Attempt to decode')
                 decoded_data = json_decoder(response.text)
                 return {'data': decoded_data}
-            else:
-                self.log.error('Unknown format for JSONDecodeError - %s',
-                               err_str)
-                raise error
+            self.log.error('Unknown format for JSONDecodeError - %s', err_str)
```

Comment:
convert to f-string?
```suggestion
            self.log.error(f"Unknown format for JSONDecodeError - {err_str}")
```

Reply:
I think from linting perspective we do not need to convert to f string within logging messages. I'll verify.